src/extensions/replay/sessionrecording.ts

```diff
@@ -345,6 +360,13 @@ export class SessionRecording {
         this.windowId = windowId
 
         this.buffer = this.clearBuffer()
+
+        if (this.sessionIdleThresholdMilliseconds >= this.sessionManager.sessionTimeoutMs) {
+            logger.warn(
+                LOGGER_PREFIX +
+                    ` session_idle_threshold_ms (${this.sessionIdleThresholdMilliseconds}) is greater than the session timeout (${this.sessionManager.sessionTimeoutMs}). Session will never be detected as idle`
+            )
+        }
```

Comment:
while testing this I discovered it was valid to set your max threshold to less than the idle threshold

you'd never go idle

and maybe you'd sit wondering why not 🤣 
so, we make both configurable and log when they're weird